### PR TITLE
Changing models to datamodels in many step modules

### PR DESCRIPTION
**Pull request by hbushouse**
There were a lot of step modules that were still using instances of "models" instead of "datamodels".

### CODE REVIEW

**Comment by hbushouse** [comment]
@STScI-JWST/maintainers Can someone have a quick look to verify that all the relative imports have been done correctly, especially instances of things like "from ..datamodels import blee" and "from .<step> import blee"?

**Comment by nden** [comment]
@hbushouse This work is already done in #58 so I suggest we close this PR. I need a bit more time to sort out the tests and then we can merge #58 without me having to resolve conflicts all the time.

**Comment by nden** [comment]
Or keep it open for now and I'll take a look after #58 is merged to see if something was missed there.

**Comment by hbushouse** [comment]
@nden It looked to me like #58 only included things in assign_wcs and a lot of built-in tests in stpipe. I didn't see any changes to all the other step modules like refpix, fringe, lastframe, straylight, extract_1d, etc.

**Comment by nden** [comment]
@hbushouse Oh you are right, sorry. It includes only things caught by the tests.

File: jwst/cube_build/cube_build.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import math
 import json
 from astropy.io import fits
-from jwst import datamodels
+from .. import datamodels
 from jwst.assign_wcs import nirspec
```
Comment (nden) [inline jwst/cube_build/cube_build.py]: from ..assign_wcs import nirspec

File: jwst/dq_init/dq_init_step.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env python
 
 from jwst.stpipe import Step
-from jwst import datamodels
+from .. import datamodels
 from jwst.dq_init import dq_initialization
```
Comment (nden) [inline jwst/dq_init/dq_init_step.py]: from . import dq_initialization

File: jwst/extract_1d/extract_1d_step.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 import os
 from jwst.stpipe import Step
```
Comment (nden) [inline jwst/extract_1d/extract_1d_step.py]: from ..stpipe .....

File: jwst/extract_2d/extract_2d.py [hunk 4]
```diff
@@ -7,7 +7,7 @@
 import copy
 import numpy as np
 from astropy.modeling import models as astmodels
-from jwst import datamodels
+from .. import datamodels
 from asdf import AsdfFile
 from jwst.assign_wcs import nirspec
```
Comment (nden) [inline jwst/extract_2d/extract_2d.py]: from ..assign_wcs import nirspec

**Comment by hbushouse** [comment]
Picky, picky, picky ...  🎱 

I'll fix 'em. Gives me something to work on during my 3-hour meeting.

**Comment by hbushouse** [comment]
Closing this PR due to a lot of conflicts with recent PEP8 changes in other merges and overall trouble with rebasing. Will try again from a clean slate.

**Pull request closed without merging**